android/app/src/main/java/org/fossasia/openevent/adapters/SessionSpeakerListAdapter.java

```diff
@@ -45,14 +47,19 @@ public RecyclerViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     public void onBindViewHolder(RecyclerViewHolder holder, final int position) {
         final Speaker current = getItem(position);
 
-        String thumbnail = current.getThumbnail();
+        String thumbnail = Utils.getSpeakerImageUri(current.getThumbnail());
+        if (thumbnail == null)
+            thumbnail = Utils.getSpeakerImageUri(current.getPhoto());
```

Comment:
@arp95 In most of the samples, only photo field is filled in and not the thumbnail one. And we want to show the assets one. So, if we get thumbnail to be null, then we show the photo (present in assets)